.travis.yml

```diff
@@ -119,6 +116,9 @@ install:
       cabal new-update head.hackage -v
     fi
   - grep -Ev -- '^\s*--' ${HOME}/.cabal/config | grep -Ev '^\s*$'
+  - (cd /tmp && echo '' | cabal new-repl -w ${HC} --build-dep fail)
+  - if [ $HCNUMVER -ge 80000 ]; then cabal new-install -w ${HC} -j2 --symlink-bindir=$HOME/.local/bin doctest --constraint='doctest ==0.16.*'; fi
```

Comment:
> `if [ $HCNUMVER -ge 80000 ]`

I'm rather uncomfortable with hard-coding the GHC version to an old release like this.

Reply:
To be clear, I'm not suggesting that we run this on every version of GHC, but only on the latest release.

Reply:
it's `-ge` so it will run on all versions `>= 8.0` (currently doctests are run for all versions)

Reply:
Ah, that makes this slightly more tolerable, then. I can live this slightly suboptimal state of affairs.